moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/moveit_cpp.h

```diff
@@ -1,216 +0,0 @@
-/*********************************************************************
```

Comment:
Pretty sure that you shouldn't delete this file, is this an accident?

Reply:
Can you format the code correctly with pre-commit? You find instructions on how to do this here: https://moveit.ros.org/documentation/contributing/code/#pre-commit-formatting-checks

Reply:
I must've accidentally deleted both moveit_cpp .h and .cpp, because I know I accidentally deleted the .cpp version and fixed that, so sorry about that. As for pre-commit I have ran it multiple times and it doesn't show errors, but I have cleaned up specifically where the CI was showcasing format-errors.

Reply:
:point_up: @CursedRock17 I guess you need to push these changes